lib/Constants.js

```diff
@@ -332,22 +332,27 @@ ReportType.init({
 const RequestStatus = {
   REQUESTED: {
     icon: 'concierge-bell',
+    text: 'Requested',
```

Comment:
eventually we should move this metadata into the database, but that will probably make more sense once we've had a couple of usability testing iterations on it.